Wikipedia/UI-V5/WMFRelatedSectionController.m

```diff
@@ -108,6 +122,21 @@ - (void)configureCell:(UICollectionViewCell*)cell
     }
 }
 
+- (UIViewController*)moreViewController {
+    #warning Remove SessionSingleton
+    WMFRelatedTitleListDataSource* dataSource =
```

Comment:
This will be gone in a second, once I push the commit that swaps `moreViewController` for `extendedListDataSource`.  However, the singleton is still necessary in the nearby data source which has to create its articles and doesn't (yet) have a way to get a data store passed to it.

Instead of adding extra properties to pass it down, I figured it will go away entirely once we remove the need to create articles.